introduction.asciidoc

```diff
@@ -55,7 +55,7 @@ Fortunately, the techniques to avoid creating a big ball of mud aren't complex.
 
 Encapsulation and Abstraction are tools that we all instinctively reach for,
 as programmers, even if we don't all use these exact words.  Allow us to dwell
-on them for a moment, since they are a recurring background them of the book.
+on them for a moment, since they are a recurring background of the book.
```

Comment:
```suggestion
on them for a moment, since they are a recurring background theme of the book.
```

Reply:
Theme! Thank you. _That does_ make sense now :)